ember-flight-icons/addon/components/flight-icon.hbs

```diff
@@ -1,13 +1,13 @@
 <svg 
-  class="flight-icon icon-{{@name}} {{if this.display "display-inline"}}"
+  class="flight-icon flight-icon-{{@name}} {{if this.display "flight-icon-display-inline"}}"
   ...attributes
   aria-hidden="true"
   data-test-icon
   fill="{{this.color}}"
-  height="{{this.size}}"
   id={{this.iconId}}
-  viewBox="0 0 {{this.size}} {{this.size}}"
+  height="{{this.size}}"
```

Comment:
nitpick: should the width come before the height?

I've always seen the width before the height (eg. https://www.w3.org/TR/css-sizing-3/#preferred-size-properties) and the same order happens in the viewBox (https://svgwg.org/svg2-draft/coords.html#ViewBoxAttribute)

Reply:
swapped! 👍